components/resources/quizzes/shared/SkillSelections.tsx

```diff
@@ -1,14 +1,25 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { Button } from "../../../ui/Button";
 import Progress from "./Progress";
 import SkillifyNavbar from "./SkillifyNavbar";
+
+type QuizObject = {
+  industries: string[];
```

Comment:
This shouldn't be in shared, this is specific to your quiz not all quizzes

Reply:
I think the type of quiz object should be a record of string to an array of some other type.

Try to figure out how record can help, if you get stuck we can chat this week